tests/test_client.py

```diff
@@ -147,3 +153,225 @@ def on_message(client, userdata, msg):
 
         packet_in = fake_broker.receive_packet(1)
         assert not packet_in  # Check connection is closed
+
+
```

Comment:
Seeing as some of the tests are testing the behaviour of `ws_set_options` and the instantiation of the wrapper I thought it might as well create the whole client to make sure it passed the options through correctly. It could be changed but I think it might actually be more work